src/components/AirdropWalletsCarousel.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { Dimensions, ViewStyle } from 'react-native';
+import Carousel from 'react-native-snap-carousel';
+
+import { AirdropWalletDetails } from 'app/consts';
+
+import { AirdropWalletsCarouselItem } from './AirdropWalletsCarouselItem';
+
+interface Props {
+  items: AirdropWalletDetails[];
+  styles?: ViewStyle;
+}
+
+const SCREEN_WIDTH = Dimensions.get('window').width;
+
+export class AirdropWalletsCarousel extends Component<Props> {
+  carouselRef = React.createRef<any>();
```

Comment:
shoudln't be any

Reply:
damn, copying from another parts of project, gets me each time